VideoLocker/src/main/java/org/edx/mobile/interfaces/TextResourceProvider.java

```diff
@@ -10,4 +10,10 @@
      * @return The text resource
      */
     int getTextResource();
+    /**
+     * Get the server mapping string value.
```

Comment:
This description and the name of this interface i.e. `TextResourceProvider` can be improved, but I am out of ideas.
thoughts @1zaman ?